README.md

```diff
@@ -35,19 +35,20 @@ $ dtsgen --help
   Options:
 
 
-    -V, --version     output the version number
-    --url <url>       input json schema from the url. (default: )
-    --stdin           read stdin with other files or urls.
-    -o, --out <file>  output d.ts filename.
-    -h, --help        output usage information
+    -V, --version                output the version number
+    --url <url>                  input json schema from the url. (default: )
+    --stdin                      read stdin with other files or urls.
+    -n, --namespace <namespace>  use namespace instead of definitions or components.schema from OpenAPI, or -n ~none~ to suppress namespaces.
```

Comment:
I do not prefer `~none~` value.
Please separate command option to suppress namespace.